.github/project.yml

```diff
@@ -1,4 +1,4 @@
 release:
-  current-version: "0"
```

Comment:
I'd recommend to have a separate PR to trigger the release workflow (which is triggered when this file is changed)

Reply:
Hello George, it was my intention to trigger a release. I'll revert it.

Reply:
@dcdh no need to, it can be named as you like